portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,5 @@
+var quoteNum = 0;
```

Comment:
Consider renaming this prevQuoteNum for clarity

Reply:
Renamed!